docs/defining_new_slo.md

```diff
@@ -0,0 +1,200 @@
+
+#### From Envoy access logs
+If your service is accessed by users via Envoy proxy, you can configure it to send the access logs using gRPC to the slo-exporter.
+Documentation about the slo-exporter gRPC logging module is [here](https://github.com/seznam/slo-exporter/blob/master/docs/modules/envoy_access_log_server.md),
+documentation how to configure Envoy to send the access logs over gRPC can be found [here](https://www.envoyproxy.io/docs/envoy/latest/api-v3/data/accesslog/v3/accesslog.proto).
+Complete example of the setup is [here](https://github.com/seznam/slo-exporter/tree/master/examples/envoy_proxy).
+
+#### Tailing file logs
+Similar to those previous but requires the slo-exporter to be running as a sidecar to each observed service instance and tail its application logs.
+slo-exporter documentation [here](https://github.com/seznam/slo-exporter/blob/master/docs/modules/tailer.md)
+and a full example [here](https://github.com/seznam/slo-exporter/tree/master/examples/nginx_proxy).
+
+#### Reading Kafka topic
+In some cases if you want to avoid logging and need high cardinality data you can send each event to a Kafka topic and slo-exporter would read them from there.
+Slo-exporter expects the messages to be in the particular format described [here](https://github.com/seznam/slo-exporter/blob/master/docs/modules/kafka_ingester.md).
+Full example can be found [here](https://github.com/seznam/slo-exporter/tree/master/examples/kafka).
+
+### 2. Configuring and deploying the slo-exporter
+Based on the decisions made, configure the slo-exporter according to [the documentation](https://github.com/seznam/slo-exporter/blob/master/docs/configuration.md)
+and deploy it for example in kubernetes. Example k8s manifests can be found in [../kubernetes](../kubernetes).
+
+Most common steps you will need to do in the slo-exporter pipeline are:
+  - filter out the unwanted events (e.g. [relabel module](./modules/relabel.md))
+  - normalize events' metadata (remove ids from URL etc) (e.g. [event_metadata_renamer](./modules/event_metadata_renamer.md), [relabel modules](./modules/relabel.md))
+  - build events' `event_key` (identifier used in the prometheus metrics exported by the slo-exporter for debugging) ([event_key_generator module](./modules/event_key_generator.md))
+  - classify events (in case you want to have different objectives for distinct event groups) ([metadata_classifier](./modules/metadata_classifier.md), [dynamic_classifier](./modules/dynamic_classifier.md), [statistical_classifier](./modules/statistical_classifier.md) modules)
+  - evaluate each event against given SLI thresholds - ([slo_event_producer](./modules/slo_event_producer.md) module)
+  - export events ([prometheus_exporter](./modules/prometheus_exporter.md) module)
```

Comment:
```suggestion
  1. filter out the unwanted events (e.g. [relabel module](./modules/relabel.md))
  2. normalize events' metadata (remove ids from URL etc) (e.g. [event_metadata_renamer](./modules/event_metadata_renamer.md), [relabel modules](./modules/relabel.md))
  3. build events' `event_key` (identifier used in the prometheus metrics exported by the slo-exporter for debugging) ([event_key_generator module](./modules/event_key_generator.md))
  4. classify events (in case you want to have different objectives for distinct event groups) ([metadata_classifier](./modules/metadata_classifier.md), [dynamic_classifier](./modules/dynamic_classifier.md), [statistical_classifier](./modules/statistical_classifier.md) modules)
  5. evaluate each event against given SLI thresholds - ([slo_event_producer](./modules/slo_event_producer.md) module)
  6. export events ([prometheus_exporter](./modules/prometheus_exporter.md) module)
```

On second thought here, I guess its better to pinpoint these are to be done in the particular order..